airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -147,6 +150,42 @@ def __init_subclass__(cls, **kwargs):
             if callable(value):
                 setattr(cls, attr, _check_flag_and_exit_if_server_response_error(value))
 
+    def execute_list(
+        self,
+        *,
+        path: str,
+        data_model: type[T],
+        offset: int = 0,
+        limit: int = 50,
+        params: dict | None = None,
+        **kwargs,
+    ) -> T | ServerResponseError:
+        shared_params = {**(params or {}), **kwargs}
```

Comment:
Why are `params` shoved into `shared_params` we already have `params` for that, we should either remove `params` and default `kwargs` to the extra params or remove `kwrags` as an extra param.